.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -51,14 +51,24 @@ body:
   validations:
     required: true
 
-- type: checkboxes
+- type: dropdown
   id: testing
   attributes:
     label: Tested with only WeakAuras
-    description: Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?
+    description: Did you try having WeakAuras as the only enabled addon? Or, did you run a [Bisector](https://www.curseforge.com/wow/addons/bisector) test to see if the issue is caused by another addon?
     options:
-    - label: "Yes"
-      required: true
+      - I did neither of these 😱
```

Comment:
This is the default because let's be honest, 99% of folks who clicked the box before didn't actually do what they promised they did.

Reply:
We should then strengthen the request to do so. Maybe add to bisector a mode where only an addon and its depencies are enabled, and then suggest that people use that?

E.g. 
"Some issues are caused by incompatibilities with other addons, if possible, please test whether your issue is caused by another addon. You can either manually disable all other addons or use LINK-TO-BISECTOR and INSTRUCTIONS. After testing RESTORE-INSTRUCTIONS reenables other addons. If another addon causes the issues, bisector can help you identify the problem via GUIDE-LINK"

Reply:
maybe some "force solo test" command for bisector that takes an addon name as arg and then disables everything else that the addon does not depend on, restores to default after testing is over. gives an output that we can require people pasting that includes that solo test was ran?

Reply:
Yes that what I wanted to suggest.

Reply:
To be a bit more clearer. We don't know how the current "requirement" to check the "tested with only WA" checkbox changes people's behaviour, because we only get data from those that ignored it.

From my feeling, and I might be wrong about that, we had very color picker doesn't work reports compared to how many people asked about it on the discord. So I think it's actually working.

Thus I wouldn't want to change the checkbox to be optional without directing the reporter to try without other addons. Now I think disabling all other addons is not as easy as it could be, so recommending bisector for that would be good. 

I think elvui even has an explicit mode that does disable all other addons, so maybe we should do something similar by bundling bisector. Though addon incompability isn't that common of a problem.

Reply:
I just published version 1.1.0 which provides a fast method of testing ${addonName} with all other addons disabled:

```sh
/bisect start
/bisect hint +WeakAuras # signals to Bisector that the issue involved does involve WA
/bisect hint !WeakAurasCompanion # workaround to a thing where companion addon is force-enabled by WA, but isn't a dependency of WA
/bisect continue

# after reloading, WA should be the only loaded addon (aside from Bisector & some 

# if issue persists...
debug tools like bugsack)
/bisect bad # awesome, issue is reproducible without any other addons

# or if issue vanishes...
/bisect good # there's at least one other addon needed, continue with normal algorithm

```